towhee/runtime/data_queue.py

```diff
@@ -130,6 +130,13 @@ def get_dict(self) -> Optional[Dict]:
             ret[names[i]] = data[i]
         return ret
 
+    def reset_max_size(self, max_size):
```

Comment:
consider this:
implement this method as a property setter

Reply:
done